gc/base/GCExtensionsBase.hpp

```diff
@@ -906,6 +906,10 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	MMINLINE OMR_VM* getOmrVM() { return _omrVM; }
 	MMINLINE void setOmrVM(OMR_VM* omrVM) { _omrVM = omrVM; }
 
+#if defined(OMR_GC_MODRON_SCAVENGER)
+	virtual void registerScavenger(MM_Scavenger *scavenger) { this->scavenger = scavenger; }
+	virtual void unregisterScavenger() { scavenger = NULL; }
+#endif /* OMR_GC_MODRON_SCAVENGER */
```

Comment:
See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif.
```suggestion
#endif /* defined(OMR_GC_MODRON_SCAVENGER) */
```